commit d3e70cd1c69bd775594fd2bce0994972356f5119
Author: q66 <q66@chimera-linux.org>
Date:   Mon Nov 24 14:12:07 2025 +0100

    drm_backend: add missing include
    
    ../src/backends/drm/drm_backend.cpp:234:27: error: no member named 'sleep_for' in namespace 'std::this_thread'
      234 |         std::this_thread::sleep_for(100ms);

diff --git a/src/backends/drm/drm_backend.cpp b/src/backends/drm/drm_backend.cpp
index f7189f5..a5229b6 100644
--- a/src/backends/drm/drm_backend.cpp
+++ b/src/backends/drm/drm_backend.cpp
@@ -37,6 +37,7 @@
 #include <algorithm>
 #include <cerrno>
 #include <ranges>
+#include <thread>
 #include <sys/stat.h>
 #include <unistd.h>
 // drm
